app/controllers/api/v1/projects_controller.rb

```diff
@@ -75,18 +75,22 @@ def content_from_params(ps)
   end
 
   def build_resource_for_create(create_params)
-    admin_allowed(:approved, :redirect)
+    admin_allowed(:beta_approved, :launch_approved, :redirect)
     create_params[:project_contents] = [ProjectContent.new(content_from_params(create_params))]
     add_user_as_linked_owner(create_params)
     super(create_params)
   end
 
   def build_update_hash(update_params, id)
-    admin_allowed(:approved, :redirect)
+    admin_allowed(:beta_approved, :launch_approved, :redirect)
     content_update = content_from_params(update_params)
     unless content_update.blank?
       Project.find(id).primary_content.update!(content_update)
     end
+    if update_params[:live] == false
```

Comment:
I think we'll need to add a little more state checking / modification here to ensure the projects switch between states correctly. A state machine like library helper class perhaps?
E.g. a project is 
``` JSON
{ "live": true, "beta_requested": true, "beta_approved": false, "launch_requested": false, "launch_approved": false }
```

and then an api request comes in to just set `{ launch_requested: true }`. The project states would then be 
``` JSON
{ "live": true, "beta_requested": true, "beta_approved": false, "launch_requested": true, "launch_approved": false }
```